src/pages/privacy.tsx

```diff
@@ -0,0 +1,98 @@
+import { micromark } from 'micromark'
+import { GetStaticProps } from 'next'
+
+import { LegalData, legalRepo } from './legal'
+import { Link } from '@/components/content/link'
+import { FrontendClientBase } from '@/components/frontend-client-base'
+import { renderedPageNoHooks } from '@/helper/rendered-page'
+import { BoxRenderer } from '@/serlo-editor/plugins/box/renderer'
+
+type PrivacyPageData = LegalData & { lastChange: string }
+
+export default renderedPageNoHooks<PrivacyPageData>((props) => {
+  return <Content {...props} />
+})
+
+function Content({ contentHtml, isGerman, lastChange }: PrivacyPageData) {
+  const date = new Date(lastChange).toLocaleDateString(
+    isGerman ? 'de-DE' : 'en-UK'
+  )
+  return (
+    <FrontendClientBase>
+      <h1 className="serlo-h1 mt-24">
+        {isGerman ? 'Datenschutzerklärung' : 'Privacy Policy'}
+      </h1>
+      <p className="serlo-p">
+        <i>
+          {isGerman ? 'wirksam ab dem' : 'effective'} {date}
+        </i>
+        <br />
+        {isGerman
+          ? 'Frühere Versionen findest Du im'
+          : 'You can view past versions in the'}{' '}
+        <Link
+          href={`https://github.com/serlo/serlo.org-legal/tree/main/${
+            isGerman ? 'de' : 'en'
+          }/privacy`}
+        >
+          {isGerman ? 'Archiv' : 'archive'}
```

Comment:
Shall we add this to our `i18n`?

Reply:
I opted to leave it out for now since we seldomly change it and we'll probably only have `de` and `en` versions. But not a strong opinion here.